### ISSUE TITLE
Testing with expectsEvents does not work with model events

### ISSUE THREAD

**Issue by dokimastikos** [laravel/lumen-framework#642]
- Lumen Version: 5.4.6
- PHP Version: 7.0
- Database Driver & Version: sqlite

### Description:
When I test if the event was fired, the event fired but the the expectsEvents() return "Exception: The following events were not fired [event name]"

### Steps To Reproduce:
- create event with name UserWasCreated
- Add the event to User model
```php
protected $events = [
        'created' => \App\Events\UserWasCreated::class,
];
```
- write the test
```php
public function expect_to_call_userwascreated()
{
    $this->expectsEvents(\App\Events\UserWasCreated::class);
    factory(\App\User::class)->create()
}
```
- The event will fired but test returns the following error
```
Exception: The following events were not fired: [App\Events\UserWasCreated]
```

**Comment by ismaail** [comment]
Try to change method name [fire](https://github.com/laravel/lumen-framework/blob/5.4/src/Testing/TestCase.php#L185) to `dispatch`

**Comment by ismaail** [comment]
This is fixed in 5.4.7
